alpine/Dockerfile

```
@@ -1,11 +1,12 @@
-FROM hmctspublic.azurecr.io/imported/library/node:12-alpine
+ARG NODE_VERSION=16
+FROM hmctspublic.azurecr.io/imported/library/node:$NODE_VERSION-alpine
 
 ENV APP_USER=hmcts \
   WORKDIR=/opt/app
 
 WORKDIR ${WORKDIR}
 
-RUN apk --no-cache add shadow \
+RUN apk --no-cache add shadow=4.8.1-r1 \
```

could you unpin so i can see what the error looks like? just curious and want to make sure the check is working
```suggestion
RUN apk --no-cache add shadow \
```

---

That's the linting error: https://github.com/hmcts/cnp-node-base/actions/runs/1659077974.

---

nice!